.eslintrc.json

```diff
@@ -249,19 +249,31 @@
         "wrap-regex": "error",
         "yoda": ["error", "never"],
 
-        "@typescript-eslint/class-name-casing": "error",
         "@typescript-eslint/explicit-member-accessibility": "error",
-        "@typescript-eslint/interface-name-prefix": [ "error", "always" ],
         "@typescript-eslint/member-delimiter-style": "error",
-        "@typescript-eslint/member-naming": "error",
-        "@typescript-eslint/no-angle-bracket-type-assertion": "error",
```

Comment:
Was there a rename/replacement for this rule? I may be missing something, but I'm not seeing it anywhere...

Reply:
Yes, the new rule is `@typescript-eslint/consistent-type-assertions`. This one was replace in v2.
https://github.com/typescript-eslint/typescript-eslint/releases/tag/v2.0.0